### ISSUE TITLE
WCAG: Page title - Improve how we handle titles in our apps

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#1646]
### Description

Today we create a standard `<title>` based on app name and app owner. This value do not change when the user navigates trough our forms. We should improve how this is handled in our apps. 

Example on how it is today
`frontend-test • Testdepartementet`

What do we append to this to tell the user some valuable information about that page? I guess the most relevant value we have is name the app developer sets on the page. It would also make sense to have the same name in page navigation component as the title. 

For instance in our test app the second page of the first form would be
`Summary • frontend-test • Testdepartementet`

## Announcing title on new page
We should mimic how a new page ussually works. For our users it appears they are browsing web pages as any other place, but the functionality in the browser do not support this. 
There are some info on how to handle page titles here: https://saavutettavuusmalli.hel.fi/en/saavutettavuus-palvelukehityksessa/toteutus-ja-ohjelmistotestaus/single-page-applications-spas-notes-on-accessibility/

### Tasks

- [ ] Do we have values we can use for signing, confirmation, receipt and other pages that are not made directly by the app developer?
- [ ] Look into posibility for the app developer to use custom `title` for each page 

### Acceptance Criterias

# Application Routes

- [ ] `<title>` should be unique for each page
- [ ] Titles should match spec in confluence: https://digdir.atlassian.net/wiki/spaces/BTA/pages/2815426563/URL+arkitektur+og+sidetitler

**Comment by benteSSB** [comment]
@Febakke: Fint om du tagger denne med "org/ssb"

**Comment by olavflar** [comment]
Related to https://github.com/Altinn/app-frontend-react/issues/97
https://github.com/Altinn/app-frontend-react/issues/1925

**Comment by olemartinorg** [comment]
Pages don't always have translations/texts, it's not something we've enforced before. So doing this is a breaking change of sorts, and possibly something we should look into when implementing the new page navigation.
